.circleci/docker/common/install_android.sh

```diff
@@ -10,7 +10,7 @@ apt-get autoclean && apt-get clean
 rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 pushd /tmp
-curl -Os --retry 3 https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK}-linux-x86_64.zip
+wget https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK}-linux-x86_64.zip
```

Comment:
Why do you need to replace curl with wget?

Reply:
Because curl is failing with `curl: (56) GnuTLS recv error (-54): Error in the pull function.`, I don't know the root cause of this.